hueplusplus/Hue.cpp

```diff
@@ -292,6 +292,35 @@ std::vector<std::reference_wrapper<HueLight>> Hue::getAllLights()
 	return result;
 }
 
+bool Hue::lightExists(int id)
+{
+	auto pos = lights.find(id);
+	if (pos != lights.end())
+	{
+		return true;
+	}
+	refreshState();
```

Comment:
Is it possible that a light was removed but still exists in the map? In that case, refreshState should be moved to the top

Reply:
Fixed with e98832b